src/jni_core/src/lib.rs

```diff
@@ -823,6 +828,49 @@ pub extern "system" fn Java_com_risingwave_java_binding_Binding_rowClose<'a>(
     pointer.drop()
 }
 
+#[inline(never)]
```

Comment:
Do we still need this method? The jni methods are at the same crate as where to load the function pointer, so I think we don't need it now.